plugin/api.go

```diff
@@ -161,6 +161,9 @@ type API interface {
 	// DeleteChannelMember deletes a channel membership for a user.
 	DeleteChannelMember(channelId, userId string) *model.AppError
 
+	// GetUsersInChannel gets users in given channel.
```

Comment:
Can you document the minimum version required here?

```suggestion
	// GetUsersInChannel gets users in given channel.
	//
	// Minimum server version: 5.6
```